### ISSUE TITLE
Adding support for DASCH

### ISSUE THREAD

**Issue by pkgw** [astropy/astroquery#2898]
I'm currently working on [DASCH], the project to digitize Harvard's collection of astronomical photographic glass plates. Historically DASCH has had a, frankly, very poor data access system, and one of my goals is to fix that. As part of that, I would like to contribute an astroquery module adding support for the DASCH data services.

[DASCH]: https://dasch.cfa.harvard.edu/

A lot of the DASCH data products are fairly typical: we have a bunch of images, from which sources have been extracted, from which we generate a catalog and timeseries light curves. One thing that might be a bit unusual is that we also have a database of plates. Plates are scanned to produce images, but not all plates are scanned, or some are scanned but can't be solved astrometrically, etc. Would recommended approach basically be to implement everything in one big module, or should things like this be split up?

I see your API specification, template module, etc., and I'd plan to start working from those. Would you like to see a PR with early work, or should I wait until I feel like any work is fairly polished? Any other requests or recommendations?

**Comment by keflavich** [comment]
Yes, we'd definitely like to see your work in progress - just mark the PR as a draft.  That's the easiest way for us to give feedback.

Everything should be in one module.  The catalog & image query tools should follow our API.  The unscanned/unsolved plates should have their own unique function, but it should live in the same module.

**Comment by bsipocz** [comment]
And if DASCH has any VO backend, it would be best to utilize those under the hood (we're switching over more and more modules to use TAP/SIA/etc under the hood in astroquery)
